source/features/parse-backticks.tsx

```diff
@@ -11,21 +11,24 @@ function init(): void {
 		'.commit-desc', // `isCommit`, `isCommitList`, `isRepoTree`
 		'.commit-message', // Pushed commits in `isPRConversation`, `isCompare`, `isReleasesOrTags`
 		'.message', // `isCommitList`, `isRepoTree`, `isBlame`
-		'.Box--condensed .link-gray[href*="/commit/"]', // `isSingleFile`
+		'.repository-content .js-details-container .link-gray[href*="/commit/"]', // `isSingleFile`
+		'.repository-content .js-details-container pre', // `isSingleFile`
 		'[aria-label="Issues"][role="group"] .js-navigation-open', // `isConversationList`
 		'[id^=ref-issue-]', // Issue references in `isIssue`, `isPRConversation`
 		'[id^=ref-pullrequest-]', // PR references in `isIssue`, `isPRConversation`
 		'.TimelineItem-body > del, .TimelineItem-body > ins', // Title edits in `isIssue`, `isPRConversation`
-		'.js-pinned-issue-list-item > span', // Pinned Issues
+		'.js-pinned-issue-list-item > .d-block', // Pinned Issues
 		'.pulse-section li', // `isPulse`
-		'.issues_labeled .text-gray-dark > a', // `isDashboard` "help wanted" event titles
```

Comment:
`.issues_labeled` no longer exists

Reply:
Is does when `hide-useless-newsfeed-events` is not enabled

Reply:
I don't see `.issues_labeled` appeared in `hide-useless-newsfeed-events`'s code. How are they related?

Reply:
![image](https://user-images.githubusercontent.com/16872793/87878900-32fe1f80-c9b5-11ea-9e03-f82fd5fb3db9.png)


```html
<div class="d-flex border-bottom py-3">
  <span class="mr-3"><a class="d-inline-block" data-hydro-click="{&quot;event_type&quot;:&quot;news_feed.event.click&quot;,&quot;payload&quot;:{&quot;event&quot;:{&quot;repo_id&quot;:55832243,&quot;actor_id&quot;:170270,&quot;public&quot;:true,&quot;type&quot;:&quot;IssuesEvent&quot;,&quot;target_id&quot;:null,&quot;id&quot;:12939275955,&quot;additional_details_shown&quot;:false,&quot;grouped&quot;:false},&quot;event_group&quot;:null,&quot;org_id&quot;:null,&quot;target_type&quot;:&quot;event&quot;,&quot;user_id&quot;:16872793,&quot;action_target&quot;:&quot;actor&quot;,&quot;originating_url&quot;:&quot;https://github.com/dashboard-feed&quot;}}" data-hydro-click-hmac="9ed037fa84bf3b5e5ae38f48c9dbf4a89ab425be6664ca521ef76ee97e96972e" data-ga-click="News feed, event click, Event click type:IssuesEvent target:actor" data-hovercard-type="user" data-hovercard-url="/users/sindresorhus/hovercard" data-octo-click="hovercard-link-click" data-octo-dimensions="link_type:self" href="/sindresorhus"><img class="avatar avatar-user" src="https://avatars0.githubusercontent.com/u/170270?s=64&amp;v=4" width="32" height="32" alt="@sindresorhus"></a></span>
  <div class="d-flex flex-column width-full">
    <div>
      <div class="d-flex flex-items-baseline mb-2">
        <div>
            <a data-hovercard-type="user" data-hovercard-url="/users/sindresorhus/hovercard" data-octo-click="hovercard-link-click" data-octo-dimensions="link_type:self" data-hydro-click="{&quot;event_type&quot;:&quot;news_feed.event.click&quot;,&quot;payload&quot;:{&quot;event&quot;:{&quot;repo_id&quot;:55832243,&quot;actor_id&quot;:170270,&quot;public&quot;:true,&quot;type&quot;:&quot;IssuesEvent&quot;,&quot;target_id&quot;:null,&quot;id&quot;:12939275955,&quot;additional_details_shown&quot;:false,&quot;grouped&quot;:false},&quot;event_group&quot;:null,&quot;org_id&quot;:null,&quot;target_type&quot;:&quot;event&quot;,&quot;user_id&quot;:16872793,&quot;action_target&quot;:&quot;actor&quot;,&quot;originating_url&quot;:&quot;https://github.com/dashboard-feed&quot;}}" data-hydro-click-hmac="9ed037fa84bf3b5e5ae38f48c9dbf4a89ab425be6664ca521ef76ee97e96972e" data-ga-click="News feed, event click, Event click type:IssuesEvent target:actor" class="link-gray-dark no-underline text-bold wb-break-all d-inline-block rgh-fullname" href="/sindresorhus">Sindre Sorhus</a>
            
            labeled
            an issue with
<--- see here --->            <span class="IssueLabel" style="background-color:#128A0C; color:#ffffff">help wanted</span>
            in
            <a data-hydro-click="{&quot;event_type&quot;:&quot;news_feed.event.click&quot;,&quot;payload&quot;:{&quot;event&quot;:{&quot;repo_id&quot;:55832243,&quot;actor_id&quot;:170270,&quot;public&quot;:true,&quot;type&quot;:&quot;IssuesEvent&quot;,&quot;target_id&quot;:null,&quot;id&quot;:12939275955,&quot;additional_details_shown&quot;:false,&quot;grouped&quot;:false},&quot;event_group&quot;:null,&quot;org_id&quot;:null,&quot;target_type&quot;:&quot;event&quot;,&quot;user_id&quot;:16872793,&quot;action_target&quot;:&quot;repo&quot;,&quot;originating_url&quot;:&quot;https://github.com/dashboard-feed&quot;}}" data-hydro-click-hmac="336f2887ea96db394483cbff88e1164ff3f3bb348d191873e6f242d9bae54a92" data-ga-click="News feed, event click, Event click type:IssuesEvent target:repo" data-hovercard-type="repository" data-hovercard-url="/sindresorhus/eslint-plugin-unicorn/hovercard" class="link-gray-dark no-underline text-bold wb-break-all d-inline-block" href="/sindresorhus/eslint-plugin-unicorn">sindresorhus/eslint-plugin-unicorn</a>
          <span class="f6 text-gray-light no-wrap ml-1">
            <relative-time datetime="2020-07-17T18:05:38Z" class="no-wrap" title="Jul 17, 2020, 2:05 PM EDT">2 days ago</relative-time>
          </span>
        </div>
      </div>
    </div>

    <div class="Box p-3 wb-break-all">
      <svg height="16" aria-label="Issue" class="octicon octicon-issue-opened open d-inline-block mt-1 float-left" viewBox="0 0 16 16" version="1.1" width="16" role="img"><path fill-rule="evenodd" d="M8 1.5a6.5 6.5 0 100 13 6.5 6.5 0 000-13zM0 8a8 8 0 1116 0A8 8 0 010 8zm9 3a1 1 0 11-2 0 1 1 0 012 0zm-.25-6.25a.75.75 0 00-1.5 0v3.5a.75.75 0 001.5 0v-3.5z"></path></svg>
      <div class="ml-4">
        <span class="f4 lh-condensed text-bold text-gray-dark">
          <a title="`unicorn/better-regex` produces unreadable regex" class="text-gray-dark" aria-label="`unicorn/better-regex` produces unreadable regex" data-hydro-click="{&quot;event_type&quot;:&quot;news_feed.event.click&quot;,&quot;payload&quot;:{&quot;event&quot;:{&quot;repo_id&quot;:55832243,&quot;actor_id&quot;:170270,&quot;public&quot;:true,&quot;type&quot;:&quot;IssuesEvent&quot;,&quot;target_id&quot;:null,&quot;id&quot;:12939275955,&quot;additional_details_shown&quot;:false,&quot;grouped&quot;:false},&quot;event_group&quot;:null,&quot;org_id&quot;:null,&quot;target_type&quot;:&quot;event&quot;,&quot;user_id&quot;:16872793,&quot;action_target&quot;:&quot;issue&quot;,&quot;originating_url&quot;:&quot;https://github.com/dashboard-feed&quot;}}" data-hydro-click-hmac="38900812d428de5d6cdf76382118f90215dbd8a07378058306b4a888ef0f2fbb" data-ga-click="News feed, event click, Event click type:IssuesEvent target:issue" href="/sindresorhus/eslint-plugin-unicorn/issues/769">`unicorn/better-regex` produces unreadable regex</a>
        </span>
        <span class="f4 text-gray-light ml-1">#769</span>
          <div class="dashboard-break-word lh-condensed mb-2 mt-1">
            <p>The rule is useful in some situations: <a class="issue-link js-issue-link" data-error-text="Failed to load title" data-id="234435881" data-permission-text="Title is private" data-url="https://github.com/xojs/xo/issues/227" data-hovercard-type="issue" data-hovercard-url="/xojs/xo/issues/227/hovercard" href="https://github.com/xojs/xo/issues/227">xojs/xo#227</a>
But counterproductive in others: <a class="issue-link js-issue-link" data-error-text="Failed to load title" data-id="611949909" data-permission-text="Title is private" data-url="https://github.com/DmitrySoshnikov/regexp-tree/issues/208" data-hovercard-type="issue" data-hovercard-url="/DmitrySoshnikov/regexp-tree/issues/208/hovercard" href="https://github.com/DmitrySoshnikov/regexp-tree/issues/208">DmitrySoshnikov/regexp-tree#208</a>

<span class="pl-pds">/http:<span class="pl-cce">\/</span><span class="pl-cce">\/</span><span class="pl-kos">[</span>^/<span class="pl-kos">]</span><span class="pl-c1">+</span><span class="pl-cce">\/</span>pull<span class="pl-cce">\/</span>commits/gi</span></p>
          </div>

          <div class="f6 text-gray mt-1">
            1 comment
          </div>
      </div>
    </div>
  </div>
</div>
```

Reply:
🤦‍♂️ I happened to not have this type of message in my feed. Then you probably can throw it into the next round of fixes (or will there be any)?

Reply:
Well maybe I'm going to improve this feature like mentioned in https://github.com/sindresorhus/refined-github/pull/3290#issuecomment-659262450
I may add this back at that time, thanks for reminding!

Reply:
That would be awesome!